src/io_hid_keys.rs

```diff
@@ -0,0 +1,16 @@
+// exports from <IOKit/hid/IOHIDKeys.h>
+
+use libc::c_int;
+use types::IOOptionBits;
+
+pub type IOHIDReportType = c_int;
+pub const kIOHIDReportTypeInput: IOHIDReportType   = 0;
+pub const kIOHIDReportTypeOutput: IOHIDReportType  = 1;
+pub const kIOHIDReportTypeFeature: IOHIDReportType = 2;
+pub const kIOHIDReportTypeCount: IOHIDReportType   = 3;
+
+pub type IOHIDManagerOptions = IOOptionBits;
+pub const kIOHIDManagerOptionNone: IOHIDManagerOptions                    = 0;
+pub const kIOHIDManagerOptionUsePersistentProperties: IOHIDManagerOptions = 1;
+pub const kIOHIDManagerOptionDoNotLoadProperties: IOHIDManagerOptions     = 2;
+pub const kIOHIDManagerOptionDoNotSaveProperties: IOHIDManagerOptions     = 4;
```

Comment:
Since these are intended to be bitflags, I think it would help communicate their intention by defining them as hex constants, similar to how they're defined in `IOHIDManager.h`:

```c
typedef enum {
    kIOHIDManagerOptionNone = 0x0,
    kIOHIDManagerOptionUsePersistentProperties = 0x1,
    kIOHIDManagerOptionDoNotLoadProperties = 0x2,
    kIOHIDManagerOptionDoNotSaveProperties = 0x4,
} IOHIDManagerOptions;
```

I don't have a really strong opinion about this, but it'd be nice to do if you're already moving the definitions to another file.